examples/src/utils/fixtures.js

```diff
@@ -91,3 +91,8 @@ export const totalSalesLocalIdentifier = "c11c27a0b0314a83bfe5b64ab9de7b89";
 export const totalCostsLocalIdentifier = "aaQHncjzfrtR";
 export const numberOfRestaurantsIdentifier = "aawAq8YqhM3o";
 export const averageRestaurantDailyCostsIdentifier = "aaQJzQzoeKwZ";
+export const cityCoordinatesUri = `/gdc/md/${demoProjectId}/obj/9459`;
+export const populationUri = `/gdc/md/${demoProjectId}/obj/9466`;
+export const densityUri = `/gdc/md/${demoProjectId}/obj/9467`;
+export const stateNamesUri = `/gdc/md/${demoProjectId}/obj/9462`;
+export const cityNamesUri = `/gdc/md/${demoProjectId}/obj/9460`;
```

Comment:
And we should move these new uris to `./examples/src/utils/fixturesGeoChart.js`
Also the mapbox token var so that we can remove this line on every example
```
const MAPBOX_TOKEN = process.env.EXAMPLE_MAPBOX_ACCESS_TOKEN;
```

Reply:
fixed